internal/pkg/agent/application/paths/paths.go

```diff
@@ -2,32 +2,25 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-//go:build !darwin && !windows
-
 package paths
 
-const (
-	// BinaryName is the name of the installed binary.
-	BinaryName = "elastic-agent"
-
-	// DefaultBasePath is the base path used by the install command
-	// for installing Elastic Agent's files.
-	DefaultBasePath = "/opt"
+import "path/filepath"
 
-	// ServiceName is the service name when installed.
-	ServiceName = "elastic-agent"
+const (
 
-	// ShellWrapperPath is the path to the installed shell wrapper.
-	ShellWrapperPath = "/usr/bin/elastic-agent"
+	// defaultAgentVaultName is keychain item name for keychain based vault (available on MacOS at the moment)
+	defaultAgentVaultName = "co.elastic.elastic-agent"
 
-	// ShellWrapper is the wrapper that is installed.  The %s must
-	// be substituted with the appropriate top path.
-	ShellWrapper = `#!/bin/sh
-exec %s/elastic-agent $@
-`
+	// defaultAgentVaultPath is the directory name where the file-based vault store is located
```

Comment:
[Suggestion]
I'd say the vault isn't a store, besides the agent already have a state store. So to avoid confusion, I'd suggest to remove the `store` here.


```suggestion
	// defaultAgentVaultPath is the directory name where the file-based vault is located
```